Packs/cyberark_AIM/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### CyberArk AIM
+- Added new version of the ***CyberArk AIM*** integration as ***CyberArk AIM v2***.
+- The ***CyberArk AIM v2*** integration supports Windows authentication and certificate-based authentication.
+- The ***CyberArk AIM*** integration is still supported.
```

Comment:
maybe we should split it to 

##### CyberArk AIM v2

##### CyberArk AIM

@kirbles19 WDYT?

Reply:
We should only write for the v2 if there are no changes for the v1.